tests/test_get_keepbits.py

```diff
@@ -29,3 +30,42 @@ def test_get_keepbits_inflevel_dim(rasm_info_per_bit, inflevel):
     if isinstance(inflevel, (int, float)):
         inflevel = [inflevel]
     assert (keepbits.inflevel == inflevel).all()
+
+
+def test_get_keepbits_informationFilter():
+    ds = xr.tutorial.load_dataset("air_temperature")
+    info = xb.get_bitinformation(ds, dim="lat")
+    var = info["air"]
+    for i in range(var.size):
+        if i >= 19 and i <= 24:
+            var[i] = 0.05
+    keepbits_dataset = get_keepbits(
+        info,
+        inflevel=[0.90],
+        information_filter="On",
+        **{"threshold": 0.7, "tolerance": 0.001}
+    )
+    keepbits = keepbits_dataset["air"].values
+    assert keepbits == 5
+
+
+def test_get_keepbits_informationFilter_1():
```

Comment:
Could you please add a doctoring to this test as well? I guess you are testing here whether the keepbits stay the same after applying the filter? That's great! The `air_temperature` dataset assumably does not have any artificial information, so I am missing one additional test, where the application of the filter actually does make a difference. This could potentially be integrated into your above's test.